### PR TITLE
New slide download command

### PR DESCRIPTION
**Pull request by matm**
Giving a valid slide ID, will download the raw file image.

### CODE REVIEW

File: cmd/slidedownload.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		}
+		defer out.Close()
+		fmt.Printf("Downloading image %q, %d bytes to write\n", fname, resp.ContentLength)
+
+		bar := pb.New64(resp.ContentLength).SetUnits(pb.U_BYTES)
+		bar.Start()
+
+		// Use proxy reader to deal with the progress state
+		reader := bar.NewProxyReader(resp.Body)
+		io.Copy(out, reader)
```
Comment (tbruyelle) [inline cmd/slidedownload.go]: Really cool library :+1:

File: cmd/slidedownload.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		info, err := api.SlideInfo(id)
+		if err != nil {
+			log.Fatal(err)
+		}
+		resp, err := api.DownloadSlide(id)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fname := info["filename"].(string)
+		out, err := os.Create(fname)
```
Comment (tbruyelle) [inline cmd/slidedownload.go]: Should we check if the file doesn't already exist before override it ?

Comment (matm) [reply cmd/slidedownload.go]: We could do that indeed. Working on it, ty.

Comment (matm) [reply cmd/slidedownload.go]: Fixed in 611310d

**Pull request merged**